pages/integrations/index.tsx

```diff
@@ -2,19 +2,47 @@ import Head from "next/head";
 import Link from "next/link";
 import prisma from "../../lib/prisma";
 import Shell from "../../components/Shell";
-import { useEffect, useState } from "react";
+import { useEffect, useState, useRef } from "react";
 import { getSession, useSession } from "next-auth/client";
 import { CheckCircleIcon, ChevronRightIcon, PlusIcon, XCircleIcon } from "@heroicons/react/solid";
 import { InformationCircleIcon } from "@heroicons/react/outline";
 import { Switch } from "@headlessui/react";
 import Loader from "@components/Loader";
 import classNames from "@lib/classNames";
 import { Dialog, DialogClose, DialogContent, DialogHeader, DialogTrigger } from "@components/Dialog";
+import AddCalDavIntegration from "@lib/integrations/CalDav/components/AddCalDavIntegration";
+import request from "@lib/request";
+
+type Integration = {
+  installed: boolean;
+  credential: unknown;
+  type: string;
+  title: string;
+  imageSrc: string;
+  description: string;
+};
+
+type Props = {
+  integrations: Integration[];
+};
+
+export default function Home({ integrations }: Props) {
+  const [, loading] = useSession();
 
-export default function IntegrationHome({ integrations }) {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const [session, loading] = useSession();
```

Comment:
Did you resolve conflicts here? I feel like I've seen your code (the green one) at first, then the removed (red) one coming from main (I've resolved it myself in my PR as well) and now you're bringing back the presumably old code.